examples/generator-fullstack/server/tsconfig.json

```diff
@@ -7,7 +7,7 @@
       "dom"
     ],
     "resolveJsonModule": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
Generated resolvers do not use `any` for arguments. Server fails to start:

```
"⨯ Unable to compile TypeScript:src/resolvers/generated/note.ts(5,17): error TS7006: Parameter 'parent' implicitly has an 'any' type.src/resolvers/generated/note.ts(5,25): error TS7006: Parameter 'args' implicitly has an 'any' type.src/resolvers/generated/note.ts(5,31): error TS7006: Parameter 'context' implicitly has an 'any' type.src/resolvers/generated/note.ts(9,20): error TS7006: Parameter 'parent' implicitly has an 'any' type.src/resolvers/generated/note.ts(9,28): error TS7006: Parameter 'args' implicitly has an 'any' type.src/resolvers/generated/note.ts(9,34): error TS7006: Parameter 'context' implicitly has an 'any' type.src/resolvers/generated/note.ts(16,18): error TS7006: Parameter 'parent' implicitly has an 'any' type.src/resolvers/generated/note.ts(16,26): error TS7006: Parameter 'args' implicitly has an 'any' type.src/resolvers/generated/note.ts(16,32): error TS7006: Parameter 'context' implicitly has an 'any' type.src/resolvers/generated/note.ts(27,18): error TS7006: Parameter 'parent' implicitly has an 'any' type.src/resolvers/generated/note.ts(27,26): error TS7006: Parameter 'args' implicitly has an 'any' type.src/resolvers/generated/note.ts(27,32): error TS7006: Parameter 'context' implicitly has an 'any' type."
```

Reply:
All should be strongly typed. This example in GraphQL CLI is actually very very focused on strong typing as this comes from graphql-code-generator.

Reply:
Well then we should probably change the default resolver args to use `any`

https://github.com/aerogear/graphback/blob/60da66c1e911881ab1fbac9c9854ec4b4be18ada/packages/graphback-codegen-resolvers/src/output/resolverTemplates.ts#L1